stable/traefik/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: traefik
-version: 1.29.2
-appVersion: 1.5.4
+version: 1.30.0
+appVersion: 1.6.1
```

Comment:
Originally I had not intended to change the Traefik image version here. However, it turns out that the custom ingress class support only landed in release 1.6

I can separate the bump to 1.6 into another PR if desired.